plugins/salesforce-as-datasource-as-eventsource/README.md

```diff
@@ -1 +1,146 @@
+
+Welcome to the [Godspeed](https://www.godspeed.systems/) salesforce Plugin! 🚀
+
+Salesforce is a powerful cloud-based customer relationship management (CRM) platform that enables businesses to manage their sales, marketing, and customer service operations efficiently. 
+
+A brief description of how to use Salesforce plug-in in our godspeed framework as Data Source as Event Source. 
+
+## Steps to use Salesforce plug-in in godspeed framework:
+
+## How to Use
+- Create a godspeed project from the CLI , open the created project in vscode and then add the plugin from the CLI of vscode, select the `@godspeedsystems/plugins-salesforce-as-datasource-as-eventsource` to integrate the plugin.
+
+```
+> godspeed plugin add
+       ,_,   ╔════════════════════════════════════╗
+      (o,o)  ║        Welcome to Godspeed         ║
+     ({___}) ║    World's First Meta Framework    ║
+       " "   ╚════════════════════════════════════╝
+? Please select godspeed plugin to install: (Press <space> to select, <Up and Down> to move rows)
+┌──────┬────────────────────────────────────┬────────────────────────────────────────────────────────────────────┐
+│      │ Name                               │ Description                                                        │
+├──────┼────────────────────────────────────┼────────────────────────────────────────────────────────────────────┤
+│  ◯   │ prisma-as-datastore                │ Prisma as a datasource plugin for Godspeed Framework.              │
+├──────┼────────────────────────────────────┼────────────────────────────────────────────────────────────────────┤
+│  ◯   │ aws-as-datasource                  │ aws as datasource plugin for Godspeed Framework                    │
+├──────┼────────────────────────────────────┼────────────────────────────────────────────────────────────────────┤
+│  ◯   │ excel-as-datasource                │ excel as datasource plugin for Godspeed Framework                  │
+├──────┼────────────────────────────────────┼────────────────────────────────────────────────────────────────────┤
+│  ◯   │ mailer-as-datasource               │ mailer as datasource plugin for Godspeed Framework                 │
+├──────┼────────────────────────────────────┼────────────────────────────────────────────────────────────────────┤
+│ ❯◯   │ salesforce-as-datasource-as-eventsource │ salesforce as datasource-as-eventsource plugin for Godspeed Framework   │
+└──────┴────────────────────────────────────┴────────────────────────────────────────────────────────────────────┘
+```
+
+### Example usage Datasource (Producer):
+
+1. Update configuration file based on your requirements in `Datasource/salesforce.yaml`.
+#### salesforce config ( src/datasources/salesforce.yaml )
+```yaml
+type: salesforce
+loginUrl: 'https://dummy.salesforce.com'
+username: dummy.username.password@organizationname.com
+password: password@123
+```
+
+
+
+#### salesforce event for Producer ( src/events/salesforce_pub.yaml )
```

Comment:
the title should be #### An example event which triggers a Salesforce API call ( src/events/salesforce_pub.yaml )